### PR TITLE
Added serverless API

### PR DESCRIPTION
**Pull request by milo157**
Added the ability to run this as a serverless API on Cerebrium

### CODE REVIEW

File: examples/serverless-api/cerebrium.toml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+[cerebrium.deployment]
+name = "serverless-api"
+python_version = "3.11"
+docker_base_image_url = "debian:bookworm-slim"
+include = "[./*, main.py, cerebrium.toml]"
+exclude = "[.*]"
+
+[cerebrium.hardware]
+cpu = 2
+memory = 12.0
```
Comment (rany2) [inline examples/serverless-api/cerebrium.toml:10]: Isn't 12GB RAM a bit excessive for this? I've never heard of Cerebrium but I'll try to check it out to test the PR.

Comment (milo157) [reply examples/serverless-api/cerebrium.toml:10]: Sorry you are right - thats the default when I created the project

**Comment by rany2** [comment]
Thanks for your PR! The new docs for the serverless API are well-written too. I'll merge it in a bit and might try to add support for other serverless platforms.

**Comment by milo157** [comment]
@rany2 I think I fixed the formatting issues

**Comment by rany2** [comment]
I won't merge it. I don't really want this to be used outside of personal use and I think deploying to a serverless API encourages this use-case. Microsoft is already cracking down on this library in Mainland China as it's being used heavily for commercial reasons.

Sorry about this but I've already had to deal with adding a new DRM they implemented to make it not work in Mainland China: https://github.com/rany2/edge-tts/pull/303

It's really odd that Chinese companies are using it commercially as the Azure API isn't really expensive in the first place and I've never encouraged this use-case when people asked me if it's OK to, but whatever.

I initially used it to generate e-books and later to generate short <10second MP3s to use in Home Assistant, later it became the only way to use this service on Linux because Microsoft dropped support for the TTS on Linux for some reason :')

**Comment by zhongtanru** [comment]
Edge-tts is good for teaching and learning English. We English teachers need it.

**Comment by rany2** [comment]
@zhongtanru It should work again with edge-tts version 6.1.16 :)

**Comment by zhongtanru** [comment]
So great!Many thanks! We English teachers really need it in China.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by zhongtanru** [comment]
It doesn't work today in china.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by zhongtanru** [comment]
Thank you! I will. I I compiled an application by VB.net， in order to read the dialogue of two or three roles，The application will read the text line by line according to the roles such as A:B: by calling the main python . Before version six 6.1.15，It was OK.However from the new version ，the application will not read From line one but from the line two. I try to fix my application But find nothing wrong. Its so weird.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by zhongtanru** [comment]
Thank you! I will try it. I compiled an application using VB.NET to read the dialogue between two or three roles. The application reads the text line by line according to the roles (e.g., A:B:) by calling the main Python script. Everything worked fine before version 6.1.15. However, with the new version, the application starts reading from the second line instead of the first line. I tried to fix the application but found no issues. It's really strange.



获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by zhongtanru** [comment]
I just undagted  the version 6.1.19, however,  when i packed my APP, it COULDN'T read. When I RAN by python, it CAN read. Why?

________________________________
发件人: Rany ***@***.***>
发送时间: 2024年11月11日 19:56
收件人: rany2/edge-tts ***@***.***>
抄送: zhongtanru ***@***.***>; Mention ***@***.***>
主题: Re: [rany2/edge-tts] Added serverless API (PR #282)


@zhongtanru<https://github.com/zhongtanru> I tried implementing a fix that might help related to clock skew. You can try it with edge-tts version 6.1.18.

―

**Pull request closed without merging**